archive/imo/imo2013_q5.lean

```diff
@@ -0,0 +1,426 @@
+/-
+Copyright (c) 2020 David Renshaw. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: David Renshaw
+-/
+
+import algebra.geom_sum
+import data.rat.basic
+import data.rat.order
+import data.real.basic
+
+/-!
+# IMO 2013 Q5
+This is a direct translation of the solution found in
```

Comment:
I don't know if this has been done in other IMO formalisations, but I think we should quote the problem statement in the module docstring. For the casual reader it will be very convenient to know up front what the target of this file is.

Reply:
Good point. I've expanded the docstring with the problem statement.